daal4py/sklearn/ensemble/tests/test_decision_forest.py

```diff
@@ -31,9 +31,9 @@
     import RandomForestRegressor as DaalRandomForestRegressor
 from daal4py.sklearn._utils import daal_check_version
 
-ACCURACY_RATIO = 0.85
+ACCURACY_RATIO = 0.95 if daal_check_version((2021, 'P', 400)) else 0.85
```

Comment:
I understand correctly, the accuracy has bad?

Reply:
No, accuracy and log loss in new implementation is better

Reply:
Also I checked MSE in the regression example(prediction saves in file decision_forest_regression_batch.csv), it used to ~17,65; in new implementation it's ~17,26. It means that error is decreased.